binary-quantization-mistral/bq-mistral-embed.md

```diff
@@ -0,0 +1,537 @@
+# Enhance Mistral Embed with Qdrant's Binary Quantization
+
+| Time: 60 min | Level: Advanced | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://githubtocolab.com/qdrant/examples/blob/master/binary-quantization-mistral/Mistral-Embed.ipynb)   |
+| --- | ----------- | ----------- |
+
+Mistral Embed offers state-of-the-art semantic for extracting representation of text extracts. However, the size of the embeddings are a challenge, especially with real-time search and retrieval. In this article, we explore how you can use Qdrant’s Binary Quantization to enhance the performance and efficiency of MistralAI embeddings.
```

Comment:
https://githubtocolab.com/qdrant/examples/blob/master/binary-quantization-mistral/Mistral-Embed.ipynb - the very first row - `mistral ai embeddings not popular`, is it ok to write not popular, maybe `not that popular`, or maybe we could just omit the sentence?

`Mistral Embed offers state-of-the-art semantic for extracting representation of text extracts` - it catches my eye: firstly, does it actually provide semantic`s` ? And maybe we could avoid the tautology - `extracting extracts`. 
I would say that it rather `offers state-of-the-art embeddings capturing semantics (or semantic meanings) of the text extracts`

However, the size`s` (or `is` instead of `are`)